lib/format.c

```diff
@@ -452,9 +438,8 @@ static void format_sprintf_num(void *ctx, feedfunc feed, u64 num64, u32 flags, i
 	}
 	else if (flags & FLAG_OCT) {
 		if (flags & FLAG_64BIT) {
-			int i;
 			// 30 bits
-			for (i = 0; i < 10; ++i) {
+			for (i = 0; i < 10; i++) {
```

Comment:
We prefer ++i

Reply:
And here is the answer for my question above :P I'll change it back to postincrement.

Reply:
Fixed.